ui/lib/sync/addon/routes/secrets.ts

```diff
@@ -9,23 +9,31 @@ import { hash } from 'rsvp';
 
 import type RouterService from '@ember/routing/router-service';
 import type StoreService from 'vault/services/store';
+import AdapterError from 'ember-data/adapter'; // eslint-disable-line ember/use-ember-data-rfc-395-imports
```

Comment:
I think you can do 
`import type AdapterError from '@ember-data/adapter';` (adding the `@` before the ember-data piece) and remove the es-lint-disable comment based on these [docs](https://github.com/ember-cli/eslint-plugin-ember/blob/master/docs/rules/use-ember-data-rfc-395-imports.md#examples)